utils/treesitter/testdata

```diff
@@ -0,0 +1 @@
+../../toolchain/parser/testdata
```

Comment:
added as TODO for a follow up in utils/treesitter/BUILD

Reply:
Looks like there is already a `filegroup` with the explorer's testdata. It is declared: https://github.com/carbon-language/carbon-lang/blob/trunk/explorer/BUILD#L78
and used: https://github.com/carbon-language/carbon-lang/blob/c0d18a62ebddfbfb97e64dc6b935a37abe885bf3/explorer/fuzzing/BUILD#L35

You should be able to copy that example, and just add another package to the visibility list

Reply:
I don't understand how to filter the `carbon_files` filegroup in treesitter/BUILD (to exclude strings and fail_ tests)  

closest result I found was https://groups.google.com/g/bazel-discuss/c/U98eiNCwxIA, which is unresolved

Reply:
One solution is to make a new filegroup in instead of `carbon_files` in the explorer/BUILD that does the exclusions.

Reply:
I didn't like that earlier because it felt like moving treesitter implementation detail into explorer BUILD.

Now, I think it is fine because explorer controls what testdata to give to treesitter.